### ISSUE TITLE
GetContentScaleFactor() returns 1.0 under Linux even if scaling is set to 200%

### ISSUE THREAD

**Issue by DarkFenX** [wxWidgets/Phoenix#1327]
**Operating system**: Debian Testing, affects X11 installation - cannot test on Wayland
**wxPython version & source**: 4.0.6+dfsg-1 from Debian repo
**Python version & source**: python 3.7 from debian repo

**Description of the problem**:
Content scaling is returned as 1 despite actually being 2, which leads to inability to use hi-res assets within the app.

```python
import wx


class MainFrame(wx.Frame):

    def __init__(self):
        wx.Frame.__init__(self, None, wx.ID_ANY, 'test')
        self.Maximize()
        self.Show()
        print(wx.GetApp().GetTopWindow().GetContentScaleFactor())


app = wx.App(False)
mf = MainFrame()
app.MainLoop()
```

![image](https://user-images.githubusercontent.com/275209/63144361-3e093c80-bffb-11e9-9c2b-c9081ccdc6ec.png)

**Comment by DarkFenX** [comment]
Looking through the docs, i suppose it's something which is handled by FromDIP in 3.10? https://wxpython.org/Phoenix/docs/html/wx.Window.html#wx.Window.FromDIP

**Comment by RobinD42** [comment]
The GTK3 implementation of `GetContentScaleFactor` in 3.1 is doing the right thing:

![Snap001](https://user-images.githubusercontent.com/808502/63199987-64979980-c034-11e9-9a4a-ce38f1000f68.png)

**Comment by swt2c** [comment]
Yep.  wxWidgets issue.  We're looking at trying to backport some of the HiDPI fixes to 3.0 though.

**Comment by DarkFenX** [comment]
FYI returns 2.0 starting from 4.0.7.
